packages/vitest/src/node/plugins/runnerTransform.ts

```diff
@@ -149,6 +149,7 @@ export function ModuleRunnerTransform(): VitePlugin {
           // https://github.com/vitejs/vite/pull/20502
           if (name === 'ssr') {
             delete config.ssr?.noExternal
+            delete config.ssr?.external
```

Comment:
This is to fix a following error https://github.com/vitest-dev/vitest/actions/runs/19751896072/job/56596264117?pr=9121#step:8:800

```
   FAIL  test/vite-ssr-resolve.test.ts > VitestResolver with Vite SSR config in deprecated style > handles ssr.resolve.external as true
  Error: Value is non of these types `True`, `Array<T>`,  on BindingViteResolvePluginConfig.external
```

Similar to `ssr.noExternal`, Vitest intend to fully overwrite through `environments.ssr.resolve.external`, so legacy `ssr.external` can be reset here.